NAMESPACE

```diff
@@ -1,3 +1,15 @@
-exportPattern("^[[:alpha:]]+")
+export(msgpack_format)
+export(msgpack_map)
+export(msgpack_pack)
+export(msgpack_simplify)
+export(msgpack_unpack)
+export(msgpackFormat)
+export(msgpackMap)
+export(msgpackPack)
+export(msgpackSimplify)
+export(msgpackUnpack)
+export(arrayEx)
+export(enumEx)
+import(Rcpp)
+importFrom(Rcpp,evalCpp)
```

Comment:
Here we generally only need line 14 and not 13, but that is totally small fry.  No worries, and easy to fix next time.